CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/open/OpenFileAction.java

```diff
@@ -94,8 +94,8 @@ private HelpCtx getHelpCtx() {
      */
     protected JFileChooser prepareFileChooser() {
         JFileChooser chooser = new FileChooser();
-        chooser.setCurrentDirectory(getCurrentDirectory());
-        HelpCtx.setHelpIDString(chooser, getHelpCtx().getHelpID());
+        //chooser.setCurrentDirectory(getCurrentDirectory());
```

Comment:
chooser.setCurrentDirectory(getCurrentDirectory()); this line makes file choose select directory which was used last time for opening graph file, although its not applicable in case when user is opening app for firs time but restoring it as same code is share for opening graph files afterwards